docs/API/CoreAPI.md

```diff
@@ -35,20 +37,23 @@ You will see output similar to the example below (**NOTE:** rootcommand below is
     Flags:
       -h, --help[=false]: help for peer
           --logging-level="": Default logging level and overrides, see core.yaml for full syntax
+          --test.coverprofile="coverage.cov": Done
+      -v, --version[=false]: Show current version number of fabric peer server
 
 
     Use "peer [command] --help" for more information about a command.
 
 ```
 
-The `peer` command supports several subcommands, as shown above. To
+The `peer` command supports several subcommands and flags, as shown above. To
 facilitate its use in scripted applications, the `peer` command always
 produces a non-zero return code in the event of command failure. Upon success,
 many of the subcommands produce a result on **stdout** as shown in the table
 below:
 
 Command | **stdout** result in the event of success
 --- | ---
+`peer version`     | String form of `peer.version` defined in [core.yaml](https://github.com/hyperledger/fabric/blob/master/peer/core.yaml)
```

Comment:
"`peer version`     | String form of `peer.version` defined in [core.yaml]" should be "`version`     | String form of `peer.version` defined in [core.yaml]"